README.md

```diff
@@ -8,8 +8,16 @@ gem install wwwtxt
 
 ## Usage
 
+(Local)
+
+```
+bundle exec bin/wwwtxt -o "~/texts" get https://raw.githubusercontent.com/ryanmuller/www.txt/master/README.md
```

Comment:
The gemspec specifies executables in `/bin` [here](https://github.com/ryanmuller/www.txt/pull/1/files#diff-774ddac674f9ce093f3523903671cefdL17). You don't need the bundle exec, it's an installed executable gem. ie Once you install it as above...

`wwwtxt -o "~/texts" get https://raw.githubusercontent.com/ryanmuller/www.txt/master/README.md`